docs/rules/no-negated-condition.md

```diff
@@ -0,0 +1,65 @@
+# Disallow use of negated expressions in conditions (no-negated-condition)
+
+Checks against the use of a negated expression in an if condition when the else branch is empty or in a ternary operator.
```

Comment:
@ilyavolodin You are right. What about this? "Checks against the use of a negated expression in a ternary operator or in an if condition when the else branch is not empty."